lib/classes/Error.js

```diff
@@ -51,6 +53,11 @@ module.exports.SError = module.exports.ServerlessError;
 
 module.exports.logError = (e) => {
   try {
+    // console.log(e)
+    // throw errors to sentry
+    // if (!fileExistsSync(statsDisabledFilePath)) { // ignore if tracking disabled
+    errorReporter.captureException(e);
```

Comment:
Is there any way to ensure that this won't take forever when using a slow internet connection?

We've added a timeout in the `logStat` method here so that a failed attempt won't harm DX --> https://github.com/serverless/serverless/blob/754559296ec01252eeb0c7fc7132620fe93f6e36/lib/classes/Utils.js#L177